playwright/tests/shippingMethods.spec.ts

```diff
@@ -118,3 +120,36 @@ test("TC: SALEOR_35 Delete a single shipping rate from its details page @shippin
   await expect(shippingMethodsPage.weightBasedRatesSection).toContainText("No shipping rates found");
   await expect(shippingMethodsPage.weightBasedRatesSection).not.toContainText(weightBasedRate);
 });
+
+test("TC: SALEOR_36 Delete shipping zones in bulk @shipping-method @e2e", async ({
+                                                                                   page,
+                                                                                 }) => {
+
+  const shippingZone1 = SHIPPING_METHODS.shippingMethodToBeBulkDeleted1.name;
+  const shippingZone2 = SHIPPING_METHODS.shippingMethodToBeBulkDeleted2.name;
+  const shippingZone3 = SHIPPING_METHODS.shippingMethodToBeBulkDeleted3.name;
+
+  await shippingMethodsPage.gotoListView();
+  await shippingMethodsPage.basePage.checkListRowsBasedOnContainingText([shippingZone1, shippingZone2, shippingZone3]);
```

Comment:
Please refer to other page for details. ShippingMethodsPage should be extended by BasePage. Then 
`shippingMethodsPage.checkListRowsBasedOnContainingText([shippingZone1, shippingZone2, shippingZone3]);` is your target.

Reply:
Fixed